tests/clientTests.py

```diff
@@ -422,7 +455,31 @@ def testGetObjectsVersion(self):
         objects = self.client.getObjects(bucketName = bucketName, version = 1)
         
         self.validateSearchObjects(objects, resources)
+                
+    def testGetBulkUnicode(self):
+        """tests getObject: unicode parameter"""
+        populateTestData(self.client, bucketName, resourceList = unicodeResources)
+
+        bucketContents = self.client.getBucket(bucketName)
+        
+        bulkGetResult = self.client.getBulk(bucketName, map(lambda obj: unicode(obj.name), bucketContents.objects))
+        
+        tempFiles = []
+        
+        availableChunks = self.client.getAvailableChunks(bulkGetResult.jobId)
+        
+        for obj in availableChunks.bulkPlan.chunks[0].objects:
+            newFile = tempfile.mkstemp()
+            tempFiles.append(newFile)
+
+            metadata_resp = self.client.getObject(bucketName, obj.name, obj.offset, bulkGetResult.jobId, newFile[1])
+        
+        for tempFile in tempFiles:
+            os.close(tempFile[0])
+            os.remove(tempFile[1])
 
+        #jobStatusResponse = self.client.getJob(bulkGetResult.jobId)
+        #self.assertEqual(jobStatusResponse.status, LibDs3JobStatus.COMPLETED)
     
```

Comment:
This test has the same problem as the metadata tests, checking the job returns an internal server error, except it seems to always occur with this test, so it's commented out for now.